validator/nodejs/postinstall-windows.js

```diff
@@ -23,7 +23,7 @@ var fs = require('fs');
 if (process.env.OS !== 'Windows_NT') {
   console./*OK*/ error(
       'postinstall-windows.js: This script is for Windows only.');
-  process.exit(1);
+  process.exit(0);
```

Comment:
Because npm will apply this script on every platform and we decide to exit when non applicable. As long as we decide to launch the script, that's not an error anymore.